app/views/users/passwords/edit.html.erb

```diff
@@ -0,0 +1,22 @@
+<h2>Change your password</h2>
```

Comment:
I want to check my understanding -- this page is just for users who have forgotten their password and can't log in, while users who are already logged in will continue to use app/users/change_password.html.erb?

Also, could you update this to more closely match the existing page? There are a few ways it deviates from our (unfortunately not-yet-documented) standards: the HTML comments are missing, the h2 is not translatable and missing the heading class, we use "my" instead of "your", we don't wrap our forms in divs, we don't use colons in our field labels, and our notes about minimum/maximum length are marked up a little differently.

This would be more like it:
```
<!--Descriptive page name, messages and instructions-->
<h2 class="heading"><%= ts("Change My Password") %></h2>
<!--/descriptions-->

<!--subnav-->
<!--/sunbanv-->

<!--main content-->
<%= form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :put }) do |f| %>
  <%= devise_error_messages! %>
  <%= f.hidden_field :reset_password_token %>
  <dl>
    <dt><%= f.label :password, ts('New password') %></dt>
    <dd><%= f.password_field :password, autofocus: true, autocomplete: 'off', "aria-describedby" => "password-field-description" %>
      <% if @minimum_password_length %>
        <p class="footnote" id="password-field-description">
          <%= ts("%{minimum} to %{maximum} characters",
                                  minimum: @minimum_password_length,
                                  maximum: ArchiveConfig.PASSWORD_LENGTH_MAX) %>
        </p>
      <% end %>
    </dd>

    <dt><%= f.label :password_confirmation, ts('Confirm new password') %></dt>
    <dd><%= f.password_field :password_confirmation, autocomplete: 'off' %></dd>

    <dt class="landmark"><%= ts('Submit') %></dt>
    <dd class="submit actions"><%= f.submit ts('Change Password') %></dd>
  </dl>
<% end %>
<!--/content-->
```

Lastly, I was having a hard time tracking this down in the code -- why does this use @minimum_password_length rather than ArchiveConfig.PASSWORD_LENGTH_MIN? Is that something that needs to be changed in the registration form and the other change password page?